packages/mobile/package.json

```diff
@@ -192,13 +192,15 @@
         "binaryPath": "android/app/build/outputs/apk/debug/app-debug.apk",
         "build": "cd android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ..",
         "type": "android.emulator",
-        "name": "Nexus_5X_API_28"
+        "name": "Nexus_5X_API_28_x86",
```

Comment:
I would have to set this dynamically somehow so devs with others emulators set up don't see an error, but I couldn't find how.